turbinia/task_manager.py

```diff
@@ -49,7 +49,17 @@
 PSQ_QUEUE_WAIT_SECONDS = 2
 
 # Define metrics
-SERVER_TASKS = Gauge('server_tasks', 'Turbinia Server Total Tasks')
+turbinia_server_tasks_total = Gauge(
+    'turbinia_server_tasks_total', 'Turbinia Server Total Tasks')
+turbinia_server_tasks_completed_total = Gauge(
+    'turbinia_server_tasks_completed_total',
+    'Total number of completed server tasks')
+turbinia_jobs_created_total = Gauge(
```

Comment:
Do we want "created" in the metric name here?  If so, should we add it to the tasks metrics for consistency?

Reply:
Done